src/pages/index.js

```diff
@@ -1,31 +1,100 @@
-import './index.css';
+import "./index.css";
 
 // import all the classes
 import Card from "../components/Card.js";
 import FormValidator from "../components/FormValidator.js";
 import Section from "../components/Section.js";
-import {initialCards, selectors, validationSettings, cardAddForm, profileEditForm}
-      from "../utils/constants.js";
+import {
+  initialCards,
+  selectors,
+  validationSettings,
+  cardAddForm,
+  profileEditForm,
+} from "../utils/constants.js";
 import PopupWithImage from "../components/PopupWithImage.js";
 import PopupWithForm from "../components/PopupWithForm.js";
-import UserInfo from '../components/UserInfo.js';
+import UserInfo from "../components/UserInfo.js";
+import Api from "../components/Api.js";
+import { PopupWithConfirmation } from "../components/popupWithConfirmation.js";
```

Comment:
* One thing hasn't been corrected https://github.com/Safiul-Alam/se_project_aroundtheus/pull/1#discussion_r1797584587 

```js
from '../components/popupWithConfirmation.js';
```


You have `PopupWithConfirmation.js`, so the import should be like this


```js
import { PopupWithConfirmation } from '../components/PopupWithConfirmation.js';


```


The app may **crash** becuase of such import. You need to import using exact **name** of the file (in `UpperCase` )